WcaOnRails/app/models/user.rb

```diff
@@ -812,10 +814,11 @@ def editable_fields_of_user(user)
     end
     if user == self
       fields += %i(
-        password password_confirmation
-        email preferred_events results_notifications_enabled
+        current_password password password_confirmation
```

Comment:
Woops, pretty sure that's something that slept through a rebase (we recently changed our way to check for user's recent authentication in #5071).